ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundlesRoute.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { useLocation, useParams } from 'react-router-dom';
+import qs from 'qs';
+import { Alert, Bullseye, PageSection, Spinner, Title } from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+import usePermissions from 'hooks/usePermissions';
+import { ClusterInitBundle, fetchClusterInitBundles } from 'services/ClustersService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import InitBundleForm from './InitBundleForm';
+import InitBundleView from './InitBundleView';
+import InitBundlesTable from './InitBundlesTable';
+
+function InitBundlesRoute(): ReactElement {
+    const { hasReadWriteAccess } = usePermissions();
+    // Pending resolution whether resources or Admin role.
+    const hasWriteAccessForInitBundles =
+        hasReadWriteAccess('Administration') && hasReadWriteAccess('Integration');
+
+    const { search } = useLocation();
+    const isCreateAction =
+        hasWriteAccessForInitBundles &&
+        qs.parse(search, { ignoreQueryPrefix: true }).action === 'create';
```

Comment:
nit: I'd assign getting the param to a variable, and then do the compound conditional, because it's easier to read. In cases where we could have other actions, like "edit", we could use a consistent pattern, too.

```suggestion
    const action =  qs.parse(search, { ignoreQueryPrefix: true }).action;
    
    const isCreateAction =
        hasWriteAccessForInitBundles && action === 'create';
```